Makefile

```diff
@@ -1,5 +1,5 @@
 KONG_BUILD_TOOLS?=4.8.0
-BASE?=centos
+BASE?=alpine
```

Comment:
Using alpine, as per the amd64 official docker image.

Reply:
Can we pass this in as a "build argument" or do we need to override the default here. I wasn't sure if we would continue to "override" the default once the artifact is in bintray.